dev/CommonStyles/AppBarButton_themeresources.xaml

```diff
@@ -421,9 +424,9 @@
                                 Grid.Column="2"
                                 Glyph="{ThemeResource AppBarButtonFlyoutGlyph}"
                                 FontFamily="{ThemeResource SymbolThemeFontFamily}"
-                                FontSize="8"
+                                FontSize="{ThemeResource AppBarButtonFlyoutFontSize}"
```

Comment:
@llongley this resource name seems innappropriate to me.  Something like AppBarButtonSubItemChevronFontSize seems better.